csharp/ql/src/semmlecode.csharp.dbscheme

```diff
@@ -705,6 +705,9 @@ local_function_stmts(
     unique int fn: @local_function_stmt ref,
     int stmt: @local_function ref);
 
+is_entry_method(
```

Comment:
Should we also mark which are the global `Stmt`s or if the entry method was generated due to global statements being present?